### PR TITLE
Modified site router to serve Angular when user isn't logged in

### PR DESCRIPTION
**Pull request by andrewconner**
@stkem

### CODE REVIEW

File: kifi-backend/modules/shoebox/angular/src/layout/nav/nav.tpl.html [hunk 1]
```diff
@@ -22,7 +22,7 @@
     </a>
   </div>
 
-  <div ng-show="mainLib.numKeeps">
+  <div ng-show="mainLib.numKeeps >= 0">
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular/src/layout/nav/nav.tpl.html:25]: @ahsu1230 This made the sidebar not show if you had no keeps in main.

File: kifi-backend/modules/shoebox/app/com/keepit/controllers/website/HomeController.scala [hunk 2]
```diff
@@ -194,14 +192,6 @@ class HomeController @Inject() (
     }
   }
 
-  def kifeeeed = HtmlAction.authenticated { request =>
```
Comment (andrewconner) [inline kifi-backend/modules/shoebox/app/com/keepit/controllers/website/HomeController.scala:197]: @stkem Time to kill this?

**Comment by andrewconner** [comment]
Need this in prod (app currently gets redirected to /). Merging, reviews appreciated.

File: kifi-backend/modules/shoebox/app/com/keepit/controllers/website/HomeController.scala [hunk 3]
```diff
@@ -228,26 +218,6 @@ class HomeController @Inject() (
...
-      val email = emailRepo.getAllByUser(user.id.get).sortBy(a => a.id.get.id).lastOption.map(_.address)
-      val friendsOnKifi = userConnectionRepo.getConnectedUsers(user.id.get).map { u =>
-        val user = userRepo.get(u)
-        if (user.state == UserStates.ACTIVE) Some(user.externalId)
-        else None
-      }.flatten
-
-      (email, friendsOnKifi)
-    }
-    Ok(views.html.website.onboarding.userRequestReceived2(
```
Comment (2is10) [inline kifi-backend/modules/shoebox/app/com/keepit/controllers/website/HomeController.scala:244]: shall we delete the template, styles, assets too?

File: kifi-backend/modules/shoebox/angular/src/layout/nav/nav.tpl.html (hunk 1 above)
Comment (ahsu1230) [inline kifi-backend/modules/shoebox/angular/src/layout/nav/nav.tpl.html:25]: woops gotcha

File: kifi-backend/modules/shoebox/app/com/keepit/controllers/website/HomeController.scala (hunk 3 above)
Comment (andrewconner) [inline kifi-backend/modules/shoebox/app/com/keepit/controllers/website/HomeController.scala:244]: Probably so. Will add to my queue.

File: kifi-backend/modules/shoebox/app/com/keepit/controllers/website/HomeController.scala (hunk 2 above)
Comment (stkem) [inline kifi-backend/modules/shoebox/app/com/keepit/controllers/website/HomeController.scala:197]: yes :-)

**Pull request merged**